practice/src/index.html

```diff
@@ -6,30 +6,67 @@
     <link rel="stylesheet" href="./css/main.css" />
     <title>Products Management</title>
   </head>
-
   <body>
+    <!-- Card 1-->
+    <div class="card card-product">
+      <div class="card-header">
+        <svg width="15" height="15">
+          <use xlink:href="./assets/images/icons/icons.svg#delete-icon" />
+        </svg>
+      </div>
+      <div>
+        <figure class="card-image">
+          <img src="#" alt="food image" />
+        </figure>
+        <div class="card-title">
+          <p class="tertiary-title card-name">Spicy seasoned seafood noodles</p>
+          <div class="main-desc card-desc">
```

Comment:
@JennyDoJD why do we need `main-desc`? Pls consider before you create new class name

Reply:
it is common class

Reply:
@JennyDoJD could you share css file contain this class?

Reply:
[https://github.com/JennyDoJD/javascript-training/blob/feature/create-text-component/practice/src/css/components/text.css](https://github.com/JennyDoJD/javascript-training/blob/feature/create-text-component/practice/src/css/components/text.css)